ArcGIS.md

```diff
@@ -1,4 +1,28 @@
-# ArgGIS #
+# ArgGIS Tutorials #
+
+## What is GIS? ##
+A geographic information system is used to display various types of data within a map-like format. GIS can connect various types of local data that display the physical geographical locations, as well as additional descriptions. 
+## ArcGIS Online ##
+Is a cloud-based software that allows you to build interactive visual maps. This is Esri’s web-based mapping platform. You will need a License through ArcGIS in order to use it and thankful we have access to ArchGIS through Penn State!
+
+## ArcGIS Pro ##
+This is Esri’s desktop application for ArcGIS. Please keep in mind the ArcGIS was made for Windows, not Macs. 
+
+## How to Create an Account and Download ArcGIS Pro: ##
+1. Go to [ArcGIS Online](https://www.esri.com/en-us/arcgis/products/arcgis-online/overview) and click the Sign In button.
+2. You will want to sign with the ArcGIS Organization’s URL section.
+    1. You will want to type in the word pennstate
```

Comment:
i'd put the keyword pennstate in blockquotes so it's easier for folks to see and copy, with tickmarks like this: `pennstate`

Reply:
Agreed w/ @am0eba-byte : Using the code blocks with the `backticks` makes it super easy to see where to copy / paste.